Add missing changelog entries for StorageEngine (#3888)
elastic/docs-content: release-notes/elastic-cloud-serverless/index.md

* Introduces the German locale for Kibana in `beta` [#236903]({{kib-pull}}236903)
* Adds an advanced option to disable filtering of file-backed volumes and CD-ROMs in the **Device Control** plugin [#236620]({{kib-pull}}236620)

> * Removes `index.dimensions` feature flag [#135402](https://github.com/elastic/elasticsearch/pull/135402)

Reviewer: ```suggestion
* Improves TSDB ingestion by hashing dimensions only once, using a new auto-populeted `index.dimensions` private index setting [#135402](https://github.com/elastic/elasticsearch/pull/135402)
```

* Adds support for package spec v3.5 [#235942]({{kib-pull}}235942)
* Adds **View in discover** button in alert details page for SLO burn rate and ES query rules [#233855]({{kib-pull}}233855)

* Implements `Delta` function for absolute change in gauges over time [#135035](https://github.com/elastic/elasticsearch/pull/135035)

> * Improves block loader for source only runtime date fields [#135373](https://github.com/elastic/elasticsearch/pull/135373)

Reviewer 2: ```suggestion
* Improves the block loader for source-only runtime date fields [#135373](https://github.com/elastic/elasticsearch/pull/135373)
```

* Adds in-product documentation for the {{esql}} `FORK` command [#236494]({{kib-pull}}236494)
* Adds **View in discover** button in alert details page for SLO burn rate and ES query rules [#233855]({{kib-pull}}233855)

* Implements `Delta` function for absolute change in gauges over time [#135035](https://github.com/elastic/elasticsearch/pull/135035)
* Improves block loader for source only runtime date fields [#135373](https://github.com/elastic/elasticsearch/pull/135373)
* Forbids metrics used in aggregations for timeseries [#135223](https://github.com/elastic/elasticsearch/pull/135223)

> * Improves block loader for source only runtime boolean fields [#135318](https://github.com/elastic/elasticsearch/pull/135318)

Reviewer: Ditto
```suggestion
```

* Removes OTLP feature flag [#135401](https://github.com/elastic/elasticsearch/pull/135401)

> * Removes `pattern_text` feature flag [#135370](https://github.com/elastic/elasticsearch/pull/135370)

Reviewer: ```suggestion
* Adds `pattern_text` field mapper in tech preview [#135370](https://github.com/elastic/elasticsearch/pull/135370)
```

* Ensures mapped fields are remembered across simulations [#233799]({{kib-pull}}233799)
* Adds time series telemetry in xpack usage [#134214](https://github.com/elastic/elasticsearch/pull/134214)

* Adds time series telemetry in xpack usage for downsampling [#134214](https://github.com/elastic/elasticsearch/pull/134214)
* Skips null metrics in {{esql}} [#133087](https://github.com/elastic/elasticsearch/pull/133087)
* Improves block loader for source only runtime fields of type long. [#134117](https://github.com/elastic/elasticsearch/pull/134117)

> * Improves block loader for source only runtime fields of type double. [#134629](https://github.com/elastic/elasticsearch/pull/134629)

Reviewer: ```suggestion
* Improves block loader for source only runtime fields of type double [#134629](https://github.com/elastic/elasticsearch/pull/134629)
```

> * Increases default limit for TS queries [#134501](https://github.com/elastic/elasticsearch/pull/134501)

Reviewer: Remove entry that doesn't have changelog
```suggestion
```

* Improves the layout of the custom URLs list in {{ml-cap}} [#231751]({{kib-pull}}231751)
* Returns 429 status code instead of 500 for timeout handlers [#133111](https://github.com/elastic/elasticsearch/pull/133111)
* Allows configuring SAML private attributes [#133154](https://github.com/elastic/elasticsearch/pull/133154)

> * Adds ordinal range encode for tsid [#133018](https://github.com/elastic/elasticsearch/pull/133018)

Reviewer 2: ```suggestion
* Adds ordinal range encode for TSID (Time Series Identifier) [#133018](https://github.com/elastic/elasticsearch/pull/133018)
```

* Adds discernible text to the refresh button on the **Streams** > **Processing** page [#225816]({{kib-pull}}225816)
* Fixes handling of context limit errors in Playground when using the Elastic Managed LLM [#225360]({{kib-pull}}225360)

> * Removes `tmp_fdt_no_mmap` feature flag [#130308](https://github.com/elastic/elasticsearch/pull/130308)

Reviewer: ```suggestion
* Forces `niofs` for `fdt tmp` file read access when flushing stored fields [#130308](https://github.com/elastic/elasticsearch/pull/130308)
```

* Introduces a new endpoint scheme for SIEM migration [#219597]({{kib-pull}}219597)
* Extends default log pattern on server side to include error information [#219940]({{kib-pull}}219940)

> * Refactors SourceProvider creation to consistently use MappingLookup [#128213](https://github.com/elastic/elasticsearch/pull/128213)

Reviewer: ```suggestion
* Refactors `SourceProvider` creation to consistently use `MappingLookup` [#128213](https://github.com/elastic/elasticsearch/pull/128213)
```

* Enables full-text search in `STATS ... WHERE` **ES|QL** queries [#220691]({{kib-pull}}220691)
* Prevents downloading trained models that are already present in other spaces and displays a warning in Machine Learning [#220238]({{kib-pull}}220238)

> * Do not respect synthetic_source_keep=arrays if type parses arrays [#127796](https://github.com/elastic/elasticsearch/pull/127796)

Reviewer: ```suggestion
* Do not respect `synthetic_source_keep=arrays` if type parses arrays [#127796](https://github.com/elastic/elasticsearch/pull/127796)
```

* Reuses shared integration policies when duplicating agent policies in Fleet [#217872]({{kib-pull}}217872)
* Enables adding badges to all list items in the side navigation except the section header [#217301]({{kib-pull}}217301)

> * Uses FallbackSyntheticSourceBlockLoader for text fields [#126237](https://github.com/elastic/elasticsearch/pull/126237)

Reviewer: ```suggestion
* Uses `FallbackSyntheticSourceBlockLoader` for text fields [#126237](https://github.com/elastic/elasticsearch/pull/126237)
```

* Filters inference connectors that lack existing endpoints in **Connectors** [#217641]({{kib-pull}}217641)
* Fixes focusability and keyboard access issues with the **Export** tab in the **Share this dashboard** modal [#217313]({{kib-pull}}217313)

> * Improves resiliency of UpdateTimeSeriesRangeService [#126637](https://github.com/elastic/elasticsearch/pull/126637)

Reviewer: ```suggestion
* Improves resiliency of `UpdateTimeSeriesRangeService` [#126637](https://github.com/elastic/elasticsearch/pull/126637)
```

* Registers a custom integrations search provider in Fleet [#213013]({{kib-pull}}213013)
* Adds support for searchAfter and PIT (point-in-time) parameters in the Get Agents List API in Fleet [#213486]({{kib-pull}}213486)

* Removes synthetic recovery source feature flag. [#122615](https://github.com/elastic/elasticsearch/pull/122615)

> * Uses FallbackSyntheticSourceBlockLoader for boolean and date fields [#124050](https://github.com/elastic/elasticsearch/pull/124050)

Reviewer: ```suggestion
* Uses `FallbackSyntheticSourceBlockLoader` for boolean and date fields [#124050](https://github.com/elastic/elasticsearch/pull/124050)
```
